dashboard/components/LoadingSpinner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="loading-spinner" :class="{ 'loading-spinner-small': small }" v-if="loading">
+		<div class="loading-spinner-rect1"></div>
+		<div class="loading-spinner-rect2"></div>
+		<div class="loading-spinner-rect3"></div>
+		<div class="loading-spinner-rect4"></div>
+		<div class="loading-spinner-rect5"></div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { defineProps } from 'vue';
```

Comment:
you can remove this line, because `defineProps` should be part of the vue runtime core now.